clients/privacy-center/cypress/e2e/show-modal.cy.ts

```diff
@@ -10,9 +10,6 @@ describe("Fides.showModal", () => {
         options: {
           isOverlayEnabled: true,
         },
-        experience: {
```

Comment:
This was a red herring and didn't affect these tests, removing `show_banner` entirely as it's not returned by the API anymore.